src/tvheadend/entity/Recording.h

```diff
@@ -172,6 +177,7 @@ namespace tvheadend
       void SetPriority(uint32_t priority) { m_priority = priority; }
 
     private:
+      uint32_t         m_enabled;
```

Comment:
If that's the case then it's fine, I somehow thought we converted to booleans in other places.